### PR TITLE
Feature left hand nav

### PR DESCRIPTION
**Pull request by DerekBurch**
This resolves more of the issues on the asana task https://app.asana.com/0/86762801523563/86770238016774.
- Adds the create/verify/join team functionality to the left hand nav
- Tweaks some paddings/sizings
- Use the batch endpoint for fetching orgs/user libraries
- Fixes org settings scrolling

Will work soon on cleaning up some of the code/styles as needed.
@cvializ @andrewconner

### CODE REVIEW

**Comment by cvializ** [comment]
Overall looks awesome, guys : D

File: kifi-backend/modules/shoebox/angular/bower.json [hunk 1]
```diff
@@ -36,13 +36,14 @@
     "underscore.string": "2.3.3",
     "airbrake-js-client": "~0.4.3",
     "svg4everybody": "~1.0.0",
-    "angular-clipboard": "~1.1.1"
+    "angular-clipboard": "~1.1.1",
+    "angular-nanoscroller": "^0.2.1"
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/bower.json:40]: what's this do? I don't see any references to it below except in the `app.js`. does it work automatically?

Comment (DerekBurch) [inline kifi-backend/modules/shoebox/angular/bower.json:40]: If you mean angular-nanoscroller it is used in the nav bar to get a custom scroller that wont look too bad on internet explorer/edge. If it ends up being too much worse than the native scrollbar we can take it out. It is loaded as `sun.scrollable` in app.js and the sources/csss are added inside gulp.

It uses https://jamesflorentino.github.io/nanoScrollerJS/ internally

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/bower.json:40]: kk, let's give it a shot

**Pull request merged**